### PR TITLE
config: enable tichi for br

### PR DESCRIPTION
**Pull request by hi-rustin**
close https://github.com/ti-community-infra/configs/issues/173

### CODE REVIEW

**Comment by hi-rustin** [comment]
/hold

- [ ] Migration SIG check config
- [ ] Give the bot admin permission
- [] @zhouqiang-cl turn off the sre-bot features

File: prow/config/external_plugins_config.yaml [hunk 1]
```diff
@@ -720,6 +746,32 @@ ti-community-blunderbuss:
...
+    pull_owners_endpoint: https://prow.tidb.io/ti-community-owners
+    max_request_count: 2
+    exclude_reviewers:
+      # Bots
+      - ti-chi-bot
+      - ti-srebot
+      # Inactive reviewers for br.
+      - amyangfei
+      - csuzhangxc
+      - glorv
```
Comment (3pointer) [inline prow/config/external_plugins_config.yaml:760]: glorv is active reviewer

File: prow/config/external_plugins_config.yaml [hunk 2]
```diff
@@ -720,6 +746,32 @@ ti-community-blunderbuss:
...
+      - GMHDBJD
+      - IANTHEREAL
+      - leoppro
+      - lonng
+      - WangXiangUSTC
+      - YuJuncens
+      - overvenus
```
Comment (3pointer) [inline prow/config/external_plugins_config.yaml:767]: overvenus is an active reviewer

File: prow/config/external_plugins_config.yaml [hunk 3]
```diff
@@ -720,6 +746,30 @@ ti-community-blunderbuss:
...
+      - csuzhangxc
+      - GMHDBJD
+      - IANTHEREAL
+      - leoppro
+      - lonng
+      - WangXiangUSTC
+      - YuJuncens
+      - july2993
+      - suzaku
+      - Little-Wallace
```
Comment (3pointer) [inline prow/config/external_plugins_config.yaml:768]: Little-Wallace is an active reviewer.

**Review by 3pointer — COMMENTED** [review]
Rest LGTM

**Pull request merged**